dataflows/processors/dumpers/file_formats.py

```diff
@@ -203,3 +203,36 @@ def write_transformed_row(self, transformed_row):
 
     def finalize_file(self):
         self.writer.write(']')
+
+
+class GeoJSONFormat(JSONFormat):
+    def __init__(self, file, schema, **options):
+        writer = file
+        writer.write('{"type": "FeatureCollection","features":')
+        writer.__first = True
+        super(GeoJSONFormat, self).__init__(writer, schema, **options)
+
+
+    def __transform_row(self, row, schema):
```

Comment:
It feels to me that this code would be better located as an override to `write_row` and not the internal `__transform_row`.
There you would be able to iterate over the keys, use `self.fields` to retrieve the field definition for each key and extract the geo-fields from the record prior to writing.

Reply:
Ah yes, you are right, I was looking for a way to retrieve the fields type and didn't realize that